cmd/main.go

```diff
@@ -25,6 +27,8 @@ func main() {
 		handleGetBalance()
 	case "getblockheight":
 		handleGetBlockHeight()
+	case "getblock":
```

Comment:
Do we need to add this functionality to this cli? I don't think anyone should be using this cli since we have the astria-go.

Reply:
I added this in for testing but can remove it.
Is it worth removing the all cli elements from this repo?

Reply:
> Is it worth removing the all cli elements from this repo?
I think we should eventually but not right now